.devcontainer/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     iqgeo:
         extends: iqgeo-common
         container_name: iqgeo_${PROJ_PREFIX:-myproj}
-        profiles: [ 'iqgeo' ]
```

Comment:
I need to figure out why my formatter is removing these spaces

Reply:
I had the `YAML Language Support by Red Hat` installed. Removing that extension fixed it. The `YAML Language Support by Red Hat` is installed by the Kubernetes extension.